tests/integration/models/reddit/test_live.py

```diff
@@ -42,3 +44,89 @@ def test_updates(self, _):
+    @mock.patch('time.sleep', return_value=None)
+    def test_invite__already_invited(self, _):
+        self.reddit.read_only = False
+        thread = LiveThread(self.reddit, 'xyu8kmjvfrww')
+        with self.recorder.use_cassette(
+                'TestLiveContributorRelationship_'
+                'test_invite__already_invited'):
+            thread.contributor.invite('nmtake')
+            with pytest.raises(APIException) as excinfo:
+                thread.contributor.invite('nmtake')
+        assert excinfo.value.error_type == 'LIVEUPDATE_ALREADY_CONTRIBUTOR'
+
+    @mock.patch('time.sleep', return_value=None)
+    def test_invite__empty_list(self, _):
+        self.reddit.read_only = False
+        thread = LiveThread(self.reddit, 'xyu8kmjvfrww')
+        with self.recorder.use_cassette(
+                'TestLiveContributorRelationship_test_invite__empty_list'):
+            thread.contributor.invite('nmtake', [])
+
+    @mock.patch('time.sleep', return_value=None)
+    def test_invite__limited(self, _):
+        self.reddit.read_only = False
+        thread = LiveThread(self.reddit, 'xyu8kmjvfrww')
+        with self.recorder.use_cassette(
+                'TestLiveContributorRelationship_test_invite__limited'):
+            thread.contributor.invite('nmtake', ['manage', 'edit'])
+
+    @mock.patch('time.sleep', return_value=None)
+    def test_invite__name(self, _):
+        self.reddit.read_only = False
+        thread = LiveThread(self.reddit, 'xyu8kmjvfrww')
+        with self.recorder.use_cassette(
+                'TestLiveContributorRelationship_test_invite__name'):
+            thread.contributor.invite('nmtake')
+
+    @mock.patch('time.sleep', return_value=None)
+    def test_invite__none(self, _):
+        self.reddit.read_only = False
+        thread = LiveThread(self.reddit, 'xyu8kmjvfrww')
+        with self.recorder.use_cassette(
+                'TestLiveContributorRelationship_test_invite__none'):
+            thread.contributor.invite('nmtake', None)
+
+    @mock.patch('time.sleep', return_value=None)
+    def test_invite__redditor(self, _):
+        self.reddit.read_only = False
+        thread = LiveThread(self.reddit, 'xyu8kmjvfrww')
+        redditor = Redditor(self.reddit,
+                            _data={'name': 'nmtake', 'id': 'll32z'})
+        with self.recorder.use_cassette(
+                'TestLiveContributorRelationship_test_invite__redditor'):
+            thread.contributor.invite(redditor)
+
+    @mock.patch('time.sleep', return_value=None)
+    def test_remove_invite__already_removed(self, _):
+        self.reddit.read_only = False
+        thread = LiveThread(self.reddit, 'xyu8kmjvfrww')
+        with self.recorder.use_cassette(
+                'TestLiveContributorRelationship_'
+                'test_remove_invite__already_removed'):
+            thread.contributor.remove_invite('t2_ll32z')
+            thread.contributor.remove_invite('t2_ll32z')
```

Comment:
I agree because I we shouldn't write the test which leads to rely on reddit's undocumented feature. But how about `test_invite__already_invited()` ? Should I remove that too because of the same reason?